test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -8872,6 +8872,39 @@ def forward(self, x):
         self.run_test(M(2, 1), (x,))
         self.run_test(M([-1, 3], [-2, -1]), (x,))
 
+    @skipIfUnsupportedMinOpsetVersion(13)
```

Comment:
Thanks for adding the tests. Can you clarify where the empty list shows up in the graph?

Reply:
The ATEN graph looks like:

graph(%x.1 : Long(10, 5, strides=[5, 1], requires_grad=0, device=cpu)):
  %17 : Double(requires_grad=0, device=cpu) = prim::Constant[value={1.1}]()
  %18 : Long(requires_grad=0, device=cpu) = prim::Constant[value={1}]()
  %19 : Bool(requires_grad=0, device=cpu) = prim::Constant[value={1}]()
  %20 : Long(requires_grad=0, device=cpu) = prim::Constant[value={6}]()
  %6 : NoneType = prim::Constant()
  %21 : Bool(requires_grad=0, device=cpu) = prim::Constant[value={0}]()
  %8 : float[] = prim::ListConstruct()
  %9 : int[] = aten::size(%x.1) # <string>:7:9
  %10 : Long(device=cpu) = aten::__getitem__(%9, %18) 
  %16 : float[] = prim::Loop(%10, %19, %8) 
    block0(%11 : int, %15 : float[]):
      %12 : float[] = aten::append(%15, %17)
      -> (%19, %12)
  %result.1 : Tensor = aten::tensor(%16, %20, %6, %21) 
  %x.7 : Tensor = aten::add(%x.1, %result.1, %18)
  return (%x.7)

We can see %8 will create an empty list before we call size() to get the tensor's shape.